src/opendr/perception/object_tracking_3d/single_object_tracking/vpit/plots.py

```diff
@@ -0,0 +1,364 @@
+# Copyright 2020-2023 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import fire
+from plotnine import (
```

Comment:
This also seems to be missing from the dependencies. Disregard if it's installed anyway.

Reply:
Updated in requirements.txt

Reply:
Updated dependencies